### PR TITLE
Incremented version to 2.2.2

### PR DESCRIPTION
**Pull request by Pelkar**
**Release notes:**

 - [ ] Feature
 - [X] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

### CODE REVIEW

**Review by DavDo — APPROVED** [review]

File: version.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#Tue Oct 31 09:50:45 UTC 2023
+#Mon Nov 06 09:53:00 UTC 2023
 dependency.coreSdk.version=1.1.9
 widgets.versionCode=72
-widgets.versionName=2.2.0
+widgets.versionName=2.2.2
```
Comment (andrews-moc) [inline version.properties:4]: Taking into account that current version is 2.2.0, should the new one be 2.2.1?

Comment (gugalo) [reply version.properties:4]: ➕

**Review by gugalo — APPROVED** [review]
What Andrew mentioned )

File: version.properties (hunk 1 above)
Comment (Pelkar) [reply version.properties:4]: Because iOS has already released 2.2.1, we decided to skip 2.2.1 and release this feature on both platforms under 2.2.2

Comment (andrews-moc) [reply version.properties:4]: Aah, yes, we discussed it on Friday, indeed.

**Review by andrews-moc — APPROVED** [review]

**Pull request merged**